lib/core/cardano-wallet-core.cabal

```diff
@@ -108,6 +109,7 @@ library
       Data.Quantity
       Network.Wai.Middleware.ServantError
       Network.Wai.Middleware.Logging
+      Test.QuickCheck.Instances.UTCTime
```

Comment:
I'd rather have that in a shared test module :s

Reply:
Done. I've moved the instance (and `newtype` wrapper) to a separate package `test-utils` that can be imported just by targets related to testing.